src/components/Modals/EditModal.js

```diff
@@ -98,7 +100,7 @@ const EditModal = (trailObj) => {
   for (let i = 0; i < types.length; i++) {
     if (trailLayerType != "N/A" && trailLayerType != types[i]) {
       dropdownOptions.push(
-        <option defaultValue={types[i]}>
+        <option defaultValue={types[i]} key={"type-" + String(i)}>
```

Comment:
nit: prefer to use es6 template literals instead of string concatenation. Fine to deploy as is 👍

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals